include/mujincontrollerclient/mujindefinitions.h

```diff
@@ -103,6 +119,7 @@ struct MUJINCLIENT_API PickPlaceHistoryItem
     unsigned long long eventTimeStampUS; ///< time that the event ocurred in us (from Linux epoch). For "picked" this is the chuck time, for "placed this is the unchuck time, for "touched" this is the time when the robot supposedly stopped touching/disturbing the object.
     std::string object_uri; ///< the object uri
     Transform objectpose; ///< 7-values in world, unit is usually mm
+    AABB localaabb; ///< AABB of object in object frame.
```

Comment:
```suggestion
    AABB localAabb; ///< AABB of object in object frame.
```
Following our [code guidelines](https://git.mujin.co.jp/dev/codeguide/-/blob/master/cpp.md#naming). 

Also see `itLocalAABB` further down.

Reply:
i want to stick to the current naming since the coding guide line is ambiguous about what to do for abbreviated word and this follows naming in other part of systems.